src/mgmt/osp-sah.ks

```diff
@@ -596,14 +593,13 @@ fi
 subscription-manager repos ${ProxyInfo} '--disable=*' --enable=rhel-7-server-rpms --enable=rhel-7-server-optional-rpms
 
 systemctl disable NetworkManager
```

Comment:
This needs to be removed since we ll now rely on networkManager

Reply:
You are right will change